hikari/impl/entity_factory.py

```diff
@@ -1532,16 +1532,26 @@ def deserialize_partial_message(  # noqa CFQ001 - Function too long
 
         message_reference: undefined.UndefinedOr[message_models.MessageReference] = undefined.UNDEFINED
         if "message_reference" in payload:
-            crosspost_payload = payload["message_reference"]
+            message_reference_payload = payload["message_reference"]
             message_reference = message_models.MessageReference(
                 app=self._app,
-                id=snowflakes.Snowflake(crosspost_payload["message_id"]) if "message_id" in crosspost_payload else None,
-                channel_id=snowflakes.Snowflake(crosspost_payload["channel_id"]),
-                guild_id=snowflakes.Snowflake(crosspost_payload["guild_id"])
-                if "guild_id" in crosspost_payload
+                id=snowflakes.Snowflake(message_reference_payload["message_id"])
+                if "message_id" in message_reference_payload
+                else None,
+                channel_id=snowflakes.Snowflake(message_reference_payload["channel_id"]),
+                guild_id=snowflakes.Snowflake(message_reference_payload["guild_id"])
+                if "guild_id" in message_reference_payload
```

Comment:
maybe these ternarys could be done outside of the class call

Reply:
Fixed :+1: